### PR TITLE
Code cleanup 

### PR DESCRIPTION
**Pull request by brayoh**
## What will this PR do?
 - Update react router from version 2  to version 3.
 - Change from using react built-in propTypes and use an independent propTypes package instead.
 - Fix some minor coding style issues noted in review.

### CODE REVIEW

File: src/routes.js [hunk 1]
```diff
@@ -1,17 +1,15 @@
 import React from "react";
-import { Router, Route, Link, IndexRoute, hashHistory, browserHistory } from 'react-router';
```
Comment (tbaraza) [inline src/routes.js:2]: I think we still need browserHistory to ensure navigation through the pages while preserving the history. 😄

Comment (brayoh) [reply src/routes.js:2]: The browserHistory prop is already passed in to the router in [index.js](https://github.com/AndelaOSP/shufflebox-frontend-react/blob/5b3405175bd7a0aac14b8e6fb2f95eb23fd8a6ff/src/index.js#L17), it was duplicated so I removed the unnecessary instance in routes.js

File: package.json [hunk 2]
```diff
@@ -1,83 +1,84 @@
...
-        "webpack": "1.13.0",
-        "webpack-dev-middleware": "1.6.1",
-        "webpack-hot-middleware": "2.10.0"
-    },
-    "devDependencies": {},
-    "repository": {
-        "type": "git",
-        "url": "https://github.com/coryhouse/pluralsight-redux-starter"
-    }
+  "name": "shuffle-box-react",
```
Comment (akabiru) [inline package.json]: Could we have this as just `shufflebox` please? 🙂

File: package.json [hunk 3]
```diff
@@ -1,83 +1,84 @@
...
+  "version": "1.0.0",
+  "description": "Shuffle Andela members to perform various events.",
```
Comment (akabiru) [inline package.json]: Please use the description in the repo desc. "Shufflebox is an application that provides random selections and groupings for social events at Andela Kenya"

File: package.json (hunk 2 above)
Comment (brayoh) [reply package.json]: Cool, I just changed the indentation I didn't change the name, it's as it is the develop branch.

File: package.json (hunk 3 above)
Comment (brayoh) [reply package.json]: Cool.

File: package.json (hunk 2 above)
Comment (akabiru) [reply package.json]: I thought since you're here. We might as well correct it. Thanks. 👍

Comment (brayoh) [reply package.json]: I've made the changes and pushed

File: src/routes.js (hunk 1 above)
Comment (tbaraza) [reply src/routes.js:2]: Cool

**Comment by austinroy** [comment]
So do I have a go ahead to merge?

**Comment by tbaraza** [comment]
Yeah

**Pull request merged**